docs/source/app-dev/app-arch.rst

```diff
@@ -136,23 +134,13 @@ The interaction of a Daml application with the ledger is inherently asynchronous
 There are several things that can fail during this time window: the application can crash, the participant node can crash, messages can be lost on the network, or the ledger may be just slow to respond due to a high load.
 
 If you want to make sure that a command is not executed twice, your application needs to robustly handle all the various failure scenarios.
```

Comment:
"all" + "the various" is slightly redundant; I would go with just "all failure scenarios".

Reply:
Changed.